experimental/traffic-portal/src/app/shared/import-json-edit-txt/import-json-edit-txt.component.html

```diff
@@ -0,0 +1,36 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<h2 mat-dialog-title>{{title}}</h2>
+<mat-dialog-content [ngClass]="{'active':dragOn}">
```

Comment:
I think we should get rid of this component/functionality entirely. I know TPv1 does it, but text editors exist and they do this job a thousand times better than we ever could, so let's not even try.

Reply:
its a text editor only, have developed a seperate reusable component for future use of silimar use cases. I think we need to get it discussed on a. huddle.